target/linux/ramips/base-files/sbin/fixup-mac-address

```diff
@@ -13,6 +13,10 @@ case $board in
 		partname=factory
 		offset=$((0xe000))
 	;;
+        witi-512)
```

Comment:
dont add an extra case for wifi-512 but ass it to the witi case.

witi|\
witi-512)
     ...
     ;;